virtual/linux-sources-build-deps/linux-sources-build-deps-1.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+DESCRIPTION="Virtual package for linux sources build dependencies"
```

Comment:
it is not virtual by defenition, it does have providers, so this probably should be moved to sys-kernel/

Reply:
moving to sys-kernel/ would made virtual/gentoo-sources collide with sys-kernel/gentoo-sources

Reply:
No, @Zlogene commented on the *virtual/linux-sources-build-deps* not *virtual/gentoo-sources*. *virtual/linux-sources-build-deps* could become *sys-kernel/linux-sources-build-deps* from this POV.

Reply:
ok yes.